### PR TITLE
Bump supported Kobo firmware version

### PR DESCRIPTION
**Pull request by davidfor**
Plus add a tag for Dropbox support. Will be adding something for that later, but, won't harm anything at the moment.

### CODE REVIEW

File: src/calibre/devices/kobo/driver.py [hunk 1]
```diff
@@ -1376,6 +1376,7 @@ class KOBOTOUCH(KOBO):
     min_forma_fwversion             = (4, 11, 11879)
     min_librah20_fwversion          = (4, 16, 13337)  # "Reviewers" release.
     min_fwversion_epub_location     = (4, 17, 13651)  # ePub reading location without full contentid.
+    min_fwversion_dropbox           = (4, 17, 13737)  # The Forma only at this point.
```
Comment (NiLuJe) [inline src/calibre/devices/kobo/driver.py]: ```(4, 18, 13737)```? ;).

Comment (kovidgoyal) [reply src/calibre/devices/kobo/driver.py]: @davidfor is it 4, 18 or 4, 17?

Comment (NiLuJe) [reply src/calibre/devices/kobo/driver.py]: The same as `max_supported_fwversion` (i.e., 4.18).

Comment (kovidgoyal) [reply src/calibre/devices/kobo/driver.py]: yeah questionis which is correct? max_supported_firmware or this?

Comment (kovidgoyal) [reply src/calibre/devices/kobo/driver.py]: Well I guess it must be the 18 according to bug https://bugs.launchpad.net/bugs/1846942

Comment (NiLuJe) [reply src/calibre/devices/kobo/driver.py]: Yeah, 4.18.

**Pull request merged**